piperider_cli/profiler/profiler.py

```diff
@@ -203,8 +203,13 @@ def _profile_column(self, table: Table, column: Column) -> dict:
             generic_type = "boolean"
             profiler = BooleanColumnProfiler(self.engine, table, column)
         else:
-            generic_type = "other"
-            profiler = BaseColumnProfiler(self.engine, table, column)
+            if self.engine.url.get_backend_name() == 'snowflake' and str(column.type).startswith('TIMESTAMP'):
```

Comment:
Is it better to move this logic to line 196?

Reply:
yeah, it can be there.
I'm afraid the statement would be too complicated at first, but it seems to be okay.
I will refine it, thanks!